db/migrate/20160510101253_change_datetime_to_date_for_check_ins.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeDatetimeToDateForCheckIns < ActiveRecord::Migration
+  def up
+    change_column :check_ins, :worked_at, :date
+  end
+
+  def down
+   change_column :check_ins, :worked_at, :datetime
+ end
+end
```

Comment:
Had to change the `worked_at` from `datetime` to just `date` because for a datetime field, rails saves date along with time(in UTC) in the database. Because it's stored in this way, we get the UTC at the end of the date, as well as the time. I've changed the `worked_at` column over to fix the issue.